### PR TITLE
[JENKINS-68645] Move Java warning EOL buttons to upper right corner

### PR DESCRIPTION
**Pull request by NotMyFault**
See [JENKINS-68645](https://issues.jenkins.io/browse/JENKINS-68645).

The change proposed moves the buttons to manage the EOL warning to the upper-right corner, to align with the button layout used in other administrative monitors.
Although, the monitor is currently inactive because Java 11 is still supported.

### Testing done

Screenshot 2023-03-17 at 13 37 26

### Proposed changelog entries

- N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7742"></a>

### CODE REVIEW

**Review by timja — APPROVED** [review]
sure... could also just delete the code.

I can't see why this merits a changelog entry though. Users will not see this for Java 11 and it would not effect anyone till Java 11 has been deprecated

**Comment by MarkEWaite** [comment]
I've been exploring the idea of a more general "deprecated thing" administrative monitor that could be used to tell users that:

* the Jenkins project no longer updates the container they are running (`blueocean`, eventually `*-centos7`)
* the Jenkins project no longer tests the operating system they are running (`Ubuntu 18`, `Alpine 3.12`, `Windows 7`, `Windows Vista`, eventually `CentOS7`, etc.)
* the Jenkins project will stop supporting the Java version they are running (when the day comes that we drop Java 11 support)

I have the action item from Platform SIG and Docs SIG meetings to create the Jenkins Enhancement Proposal to explore the idea further.  Not done yet, but hopefully can be done before Java 11 reaches end of life for the Jenkins project.

**Review by janfaracik — APPROVED** [review]

**Comment by NotMyFault** [comment]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

File: core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message.jelly
Comment (janfaracik) [inline core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message.jelly:32]: Does it need the submit-button class? Nothing stood out for a quick CMD+SHIFT+F that I did

Comment (NotMyFault) [reply core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message.jelly]: Good catch! Works well without too 👍

**Pull request merged**
